packages/ui-react/src/components/LayoutGrid/LayoutGrid.module.scss

```diff
@@ -4,5 +4,5 @@
 
 .cell {
   display: inline-block;
-  padding: 8px;
+  padding: 16px 8px;
```

Comment:
Do we also want this for mobile? I personally think we want this for tablet and up. (Maybe we discussed differently yesterday, then i agree with that decision of course)

Reply:
This is not discussed, but I think I'd prefer to keep this the same for all breakpoints.